nifi-nar-bundles/nifi-snowflake-bundle/nifi-snowflake-services/src/test/java/org/apache/nifi/snowflake/service/SnowflakeConnectionPoolTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.snowflake.service;
+
+import org.apache.nifi.processor.AbstractSessionFactoryProcessor;
+import org.apache.nifi.processor.ProcessContext;
+import org.apache.nifi.processor.ProcessSessionFactory;
+import org.apache.nifi.processor.exception.ProcessException;
+import org.apache.nifi.util.TestRunner;
+import org.apache.nifi.util.TestRunners;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+/**
+ * Set the following constants:<p>
+ * SNOWFLAKE_URL<p>
+ * SNOWFLAKE_PASSWORD<p>
+ * SNOWFLAKE_PASSWORD<p>
+ * TABLE_NAME<p>
+ */
+@Disabled("Manual test.")
```

Comment:
This annotation should be removed and the class name should be changed to end with `IT` so it will be treated as an integration test and ignored for standard unit test execution.

Reply:
There are a handful of disabled tests that should be renamed to integration tests, so we should not continue that pattern.  Disabled tests show up as warnings during builds, which is another reason to keep a clearer distinction between automated unit tests and manual integration tests.  If this test could be run in an automated fashion, that would be great, but it does not work that way as it stands.

As an alternative solution, JUnit 5 supports conditional annotations that would enable the test when environment variables or system properties are present.  Adjusting to test to read the URL and credentials from environment variables would allow keeping this as a unit test, but making it conditional on the presence of those variables.